src/app/components/carousel/carousel.component.html

```diff
@@ -1,4 +1,20 @@
-<ion-icon name="chevron-back-outline" size="large" class="cursor-pointer"></ion-icon>
-<ion-icon name="chevron-forward-outline" size="large" class="cursor-pointer"></ion-icon>
-<img src="../../../assets/black-friday-elements-assortment_23-2149074076.avif" id="image"/>
+<div id="slider">
+    <!-- <ion-icon name="chevron-back-outline" size="large" class="cursor-pointer"></ion-icon>
+    <ion-icon name="chevron-forward-outline" size="large" class="cursor-pointer"></ion-icon> -->
+    
+    <img id="slider-1" src ="../../../assets/black-friday-elements-assortment_23-2149074076.avif" class="image"/>
```

Comment:
this isn't displaying on my end
![image](https://github.com/stevehotcodes/angular_ecommerce_frontend/assets/24773276/e1a8eb0e-f552-4857-a0f1-df726230d63b)